### ISSUE TITLE
Line numbers should not start with 1

### ISSUE THREAD

**Issue by zuphilip** [zotero/scaffold#48]
On the first lines of each translator file there is the metadata and therefore the actual code which is shown in Scaffold is usually line 15. If we have these numbers synchronized, then it is much easier to find lines which are mentioned in any debug message.

**Comment by dstillman** [comment]
https://github.com/zotero/scaffold/issues/49#issuecomment-334962836 also relevant to this

**Comment by zuphilip** [comment]
There is an option for that in ACE for setting the first line number, i.e. something like
```js
_editors["code"].getSession().setOptions({
	firstLineNumber: 15
});
```
shoul work. However, it seems that the line numbers in the debug messages and in ACE are only 3 off, but they don't align to the line numbers in the actual translator file. Better alignment of these numbers will also help people using an external editor.

**Comment by dstillman** [comment]
Fixed in 2f6767d
